proprietary/tokens/README.md

```diff
@@ -146,16 +146,17 @@ Note that redefining the value of a token is a much better approach than redecla
 ## Usage in Sass
 
 The tokens can be imported as Sass variables as well.
+As they are already prefixed, the namespace that Sass would assign isn’t necessary.
 
 ```sass
-@import "@amsterdam/design-system-tokens/dist/index.scss"
+@use "@amsterdam/design-system-tokens/dist/index.scss" as *;
 ```
 
 Import the compact tokens if you need them.
 Sass will override spacious values automatically.
```

Comment:
It's not that easy I'm afraid. If you import both `index.scss` and `compact.scss` as *, you get a 'This variable is available from multiple global modules.' error. Maybe we should suggest that users just do `@use "@amsterdam/design-system-tokens/dist/index.scss";` and `index.$ams-space-xl`? And `compact.$ams-space-xl` for the compact token?

Reply:
Ouch, indeed. I’ve updated the instructions to only use the namespace for the compact tokens – I don’t think the `index` prefix adds much.